include/af/array.h

```diff
@@ -1668,10 +1668,24 @@ extern "C" {
         \returns error codes
     */
     AFAPI af_err af_is_sparse       (bool *result, const af_array arr);
+#endif
+
+#if AF_API_VERSION >= 35
     /**
-        @}
+        \brief Get nth element element from an array
+
+        \param[out] out is the nth element requested
+        \param[in] arr is the input array
+        \param[in] idx is the index of the nth element we want from the input array
+        \return \ref AF_SUCCESS if the execution completes properly
     */
+    AFAPI af_err af_get_scalar(double* real, double* imag, const af_array arr, const unsigned idx);
```

Comment:
Remove `idx`. The C++ API is using `idx == 0` and using anything else would be a problem if the array was indexed previously.